tox.ini

```diff
@@ -24,6 +24,13 @@ setenv =
 commands =
     flake8 sphinxcontrib setup.py
 
+[testenv:pkglint]
+deps=
+    twine
+commands=
+    python setup.py sdist
```

Comment:
I strongly recommend to also lint wheels because their metadata is different. I'm not even sure if require-python is included there.

Reply:
Hey, thanks, I didn't see your comment before I merged this so I'll look at that separately.

Reply:
That's alright, I commented after the merge :)

Reply:
Oh, good, I thought I hadn't been paying attention. :-) 

See #51 for the wheel linting.